The script will fetch the repository information of users and organizations defined in the separate yaml file "ot-git-details-source.yml" and generate the specific information like Repo name, Repo description and Repo in the YAML format.

VineetOps opened the pull request:
The script will fetch the repository information of users and organizations defined in the separate yaml file "ot-git-details-source.yml" and  generate the specific information link Repo name, Repo description and Repo in the YAML format.

iamabhishek-dubey committed:
Added some new projects

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

iamabhishek-dubey committed:
Made all cards clickable

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

iamabhishek-dubey committed:
Added cloud section

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

iamabhishek-dubey committed:
Updated opensource site

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

iamabhishek-dubey committed:
Added google analytics

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

iamabhishek-dubey committed:
Added druid exporter in database section

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

iamabhishek-dubey committed:
Updated druid exporter logo

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

sandy724 committed:
Added entries for restic

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Create Python_Script.yml

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

sandy724 commented on Python_Script.yml:
```diff
@@ -0,0 +1,48 @@
+
+out_file='Open_Source_output.yml'
+d={}
+source_yaml_data=input("Please Enter the source yaml file:")
```
You should not take input expect this file is already available in repository

VineetOps replied:
Included the input file into the script

sandy724 commented on Python_Script.yml:
```diff
@@ -0,0 +1,48 @@
+import requests,json, yaml
+
+out_file='Open_Source_output.yml'
+d={}
```
use proper variable name

VineetOps replied:
updated

sandy724 commented on Python_Script.yml:
```diff
@@ -0,0 +1,48 @@
+
+out_file='Open_Source_output.yml'
+d={}
+source_yaml_data=input("Please Enter the source yaml file:")
```
input_manifest

VineetOps replied:
included file into the script

sandy724 commented on Python_Script.yml:
```diff
@@ -0,0 +1,48 @@
+
+out_file='Open_Source_output.yml'
+d={}
+source_yaml_data=input("Please Enter the source yaml file:")
```
Source Yaml file should look like this
users:
    - sandy724
    - vineet
orgs:
    - opstree
    - OT-BUILDPIPER-MARKETPLACE

VineetOps replied:
used the suggested format

VineetOps committed:
Updated suggested changes

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Updated suggested changes

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Input file

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Update Python_Script.py

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Update Python_Script.py

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Script with functions and changed logic

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Script with functions and changed logic

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Updated User

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Updated for DCO Check

signed-off-by: username vineet.yadav@opstree.com
Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Updated for DCO Check

signed-off-by: username vineet.yadav@opstree.com
Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Signed-off-by: Author Name "vineet.yadav@opstree.com"

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Signed-off-by: Author Name "vineet.yadav@opstree.com"

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Signed-off-by: username <vineet.yadav@opstree.com>

The script will fetch the repository information of users and organizations defined in the separate yaml file.
Signed-off-by: username <vineet.yadav@opstree.com>
Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Signed-off-by: username <vineet.yadav@opstree.com>

The script will fetch the repository information of users and organizations defined in the separate yaml file.
Signed-off-by: username <vineet.yadav@opstree.com>
Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
DCO check Pass Signed-off-by: username <vineet.yadav@opstree.com>

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Delete Python_Script.py

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
DCO check Pass Update Signed-off-by: username <vineet.yadav@opstree.com>

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

rajatvats-devops commented on Open_Source_Input.yml:
```diff
@@ -0,0 +1,20 @@
+
+
+
+
```
Please mention proper description in the PR as we d1iscussed

rajatvats-devops commented on Open_Source_Input.yml:
```diff
@@ -0,0 +1,20 @@
+users:
```
file name should be ot-git-details-source.yml

rajatvats-devops commented on Open_Source_Input.yml:
```diff
@@ -0,0 +1,20 @@
+
+
+
+
```
Remove unnessary blank lines

rajatvats-devops commented on Yaml_Automation.py:
```diff
@@ -0,0 +1,114 @@
+import requests as github
+import json, yaml
+
+source_yaml_data = 'Open_Source_Input.yml'
```
make this variable as constant ..

rajatvats-devops commented on Yaml_Automation.py:
```diff
@@ -0,0 +1,114 @@
+            open('Output_File.yml', "w").close()
+
+
+processUsers(getDataFromYaml)
```
You should have main function in your script ..

rajatvats-devops replied:
in which these two functions get called

VineetOps committed:
Removed Unnecessary Blank Lines

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

rajatvats-devops commented on Yaml_Automation.py:
```diff
@@ -0,0 +1,114 @@
+import os
```
Please move this utility in scripts folder .

rajatvats-devops commented on Yaml_Automation.py:
```diff
@@ -0,0 +1,114 @@
+import os
+import requests as github
```
Please name this script correctly

rajatvats-devops commented on Yaml_Automation.py:
```diff
@@ -0,0 +1,114 @@
+
+        # Append all the required fetched repo info into the array
+
+        for i in range(len(repo_data)):
```
i?

rajatvats-devops commented on Yaml_Automation.py:
```diff
@@ -0,0 +1,114 @@
+
+        # Write the yaml file with the required data
+        try:
+            out = open(output_yaml_file, mode='r').read()
```
out?

VineetOps committed:
Moved python script to script directory Signed-off-by: username <vineet.yadav@opstree.com>

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Moved python script to script directory Signed-off-by: username <vineet.yadav@opstree.com>

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Moved python script to script directory Signed-off-by: username <vineet.yadav@opstree.com>

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Moved python script to script directory Signed-off-by: username <vineet.yadav@opstree.com>

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Updated directory structure in script Signed-off-by: username <vineet.yadav@opstree.com>

Signed-off-by: Vineet Yadav <vineet.yadav@opstree.com>

VineetOps committed:
Update ot-osc-repo-generator.py
